frontend/micro-ui/web/micro-ui-internals/packages/react-components/src/atoms/Table.js

```
@@ -159,156 +155,157 @@ const Table = ({
+        style={{
+          marginTop: isIpadView ? "210px" : 0,
+          marginLeft: isIpadView ? -20 : 0,
+        }}
+      >
+        <div style={{ overflowX: "auto" }} ref={tableWrapperRef}>
+          {/* Table */}
+          <div style={{ display: "inline-block", minWidth: "100%" }}>
+            <table className={className} {...getTableProps()} style={styles} ref={tableRef} {...getNoColumnBorder(noColumnBorder)}>
+              <thead>
+                {headerGroups.map((headerGroup) => (
+                  <tr {...headerGroup.getHeaderGroupProps()}>
```

Biome (1.9.4)

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)